docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -533,6 +580,40 @@ Development value: secrets/cluster-logging-int.pem
    vpn_configuration
    ```
 
+## PR E2E Only - Create the global keyvault, ADO Library Variable Group
```

Comment:
Should these steps be located in `docs/deploy-development-rp.md`, or maybe it's own document? Will this be required for everyone's PR E2Es to run going forward?

Reply:
The "prepare a shared rp development environment" is used to deploy both the shared dev environments in eastus, westeurope, australiaeast and PR E2E environment in MSIT tenant and the new tenant we are migrating to.

These steps were never documented but they are required to have a working shared environment for PR E2E to run so I figured I would add them to this doc but mark it as "PR E2E Only".

Developers don't need to perform the steps in "prepare-a-shared-rp-development-environment.md" steps in order to get a working local rp.